package.json

```diff
@@ -23,6 +23,7 @@
     "clean": "rimraf \"{apps,packages}/*/{build*,*.tsbuildinfo}\" \"coverage\" \".serverless*\"",
     "fix:format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,html,css,scss,md,mdx,yml,yaml}\"",
     "lint:format": "prettier -l \"**/*.{js,jsx,ts,tsx,json,html,css,scss,md,mdx,yml,yaml}\"",
+    "precommit": "yarn fix:format && yarn install && yarn build && yarn test",
```

Comment:
I assume this isn't automatically a hook

Reply:
i would remove this actually, if you intention is for it to be a hook then surely you can add this line to your local git config